.github/workflows/hf-xet-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Test huggingface_hub xet tests
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  hub-python-tests:
+    runs-on: ubuntu-latest
```

Comment:
should we also run on windows too?

Reply:
If we find Windows tests breaks are common enough, we can always add them in here.